src/components/Widget/components/Conversation/components/Messages/components/ImgReply/index.js

```diff
@@ -6,13 +6,16 @@ import './styles.scss';
 class ImgReply extends PureComponent {
   render() {
     const message = this.props.message;
+    const title = message.get('title');
+    const image = message.get('image');
+    const params = message.get('params');
```

Comment:
use destructuring:
```
{ title,  image, params } = this.props
```
Have you verified that it doesn't break when params or dims is undefined?

Reply:
Params are updated to be passed as props, and have a default value in case of undefined.